Rocket.Chat/Views/Subscriptions/SubscriptionUserStatusView.swift

```diff
@@ -115,4 +145,20 @@ final class SubscriptionUserStatusView: UIView {
         MainChatViewController.shared?.logout()
     }
 
+    func openAdminPanel() {
+        guard
+            let auth = AuthManager.isAuthenticated(),
+            let baseURL = auth.settings?.siteURL,
+            let adminURL = URL(string: "\(baseURL)/admin/info?layout=embedded")
+        else {
+            return
+        }
+
+        if let controller = WebViewControllerEmbedded.instantiateFromNib() {
```

Comment:
I think you can add it to a BaseNavigationController, top is staying white now... can you take a look?